store/tikv/2pc.go

```diff
@@ -159,20 +166,23 @@ func newTwoPhaseCommitter(txn *tikvTxn, connID uint64) (*twoPhaseCommitter, erro
 		return nil, errors.Trace(err)
 	}
 
+	// Convert from sec to ms
+	maxTxnTimeUse := uint64(config.GetGlobalConfig().TiKVClient.MaxTxnTimeUse) * 1000
 	commitDetail := &execdetails.CommitDetails{WriteSize: size, WriteKeys: len(keys)}
-	metrics.TiKVTxnWriteKVCountHistogram.Observe(float64(commitDetail.WriteKeys))
-	metrics.TiKVTxnWriteSizeHistogram.Observe(float64(commitDetail.WriteSize))
+	metrics.TiKVTxnWriteKVCountHistogram.Observe(float64(len(keys)))
+	metrics.TiKVTxnWriteSizeHistogram.Observe(float64(size))
```

Comment:
Is this change caused by incorrectly resolving git's conflict? This change is not part of the previous PR.

Reply:
@qw4990 PTAL

Reply:
My mistake.
Some invalid changes have been removed now, PTAL @zz-jason @MyonKeminta